backend/controllers/authController.js

```diff
@@ -0,0 +1,24 @@
+const User = require('/models/UserModels');
+
+const authController = {
+  async getUser(req, res, next) {
+    try {
+      const data = await User.findOne({ firstName });
+    } catch (e) {
+      console.log(`Error: ${e}`);
+    }
+    console.log(`Here is the data: ${data}`);
+  },
+
+  async signUp(req, res, next) {
+    try {
+      const { username } = req.body;
+      const newUser = User.create({ username });
+    } catch (e) {
+      console.log(`Error: ${e}`);
```

Comment:
Rather than logging the error to the console I would pass it to the error handling middleware using `return next({whatever you need here})`

Reply:
Thanks for the advice, I've made adjustments and recommitted.